capstone/src/main/java/com/google/step/clients/MetricsClient.java

```diff
@@ -126,32 +147,31 @@ public WeeklyPageView getCurrentPageViews(String restaurantKey) {
                 .addSort("week", Query.SortDirection.ASCENDING);
         PreparedQuery preparedQuery = dataStore.prepare(query);
         List<Entity> results = preparedQuery.asList(FetchOptions.Builder.withDefaults());
-
-        // Map to maintain a reference to each restaurantList and make searching for them easy.
-        // The map and the list share the reference to each restaurantPageViews.
-        Map<String, List<WeeklyPageView>> restaurantNameMap = new HashMap<>();
-        Map<String, String> restaurantIdMap = new HashMap<>();
+      
+        // Map to maintain a reference to all the information needed about a restaurant
+        // and make searching for them easy.
+        Map<String, RestaurantPageViewInformation> restaurantIdMap = new HashMap<>();
```

Comment:
Is there a reason this isn't a map from `long` to `RestaurantPageViewInformation`?

Reply:
Not particularly. It could be both a String or a Long.